chartmogul/api/metrics.py

```diff
@@ -2,6 +2,17 @@
 from ..resource import Resource, DataObject, _add_method
 from collections import namedtuple
 
+metrics = [
```

Comment:
Why do we need a list here?

Reply:
Yes, I understand now. Since we use the EXCLUDE unknown keys  strategy, we have to make all keys known. 

If we were using the INCLUDE strategy, we would not need this.